budgetkey_data_pipelines/pipelines/procurement/tenders/exemptions/add_publisher_urls_resource.py

```diff
@@ -15,11 +17,24 @@
              45: []}
 
 def resource_filter(resource_data, parameters):
+    max_pages = os.environ.get("EXEMPTION_PUBLISHERS_MAX_PAGES", "")
```

Comment:
I was refering to manual testing during development, not automated testing

for example, to test with different number of pages in this case, or in the case of limiting to running for only specific publishers
this allows to quickly modify the parameters from command line with environment variables

I'll fix the update method in #34 - I'll add your comment there